x/provider/types/event.go

```diff
@@ -0,0 +1,108 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ovrclk/akash/sdkutil"
+)
+
+const (
+	evActionProviderCreate = "provider-create"
+	evActionProviderUpdate = "provider-update"
+	evActionProviderDelete = "provider-delete"
+	evOwnerKey             = "owner"
+)
+
+// EventProviderCreate struct
+type EventProviderCreate struct {
+	Owner sdk.AccAddress
+}
```

Comment:
Can you add the events from this file to a new section [here](https://github.com/ovrclk/akash/blob/189a668f583e03adc47a1c0ecb70fd0ca238debf/events/publish_test.go#L16-L19)?

I think that you'll need to update [`processEvent`](https://github.com/ovrclk/akash/blob/189a668f583e03adc47a1c0ecb70fd0ca238debf/events/publish.go#L98-L100) too.